hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieTableServiceClient.java

```diff
@@ -236,8 +238,8 @@ protected HoodieWriteMetadata<O> logCompact(String logCompactionInstantTime, boo
     WriteMarkersFactory.get(config.getMarkersType(), table, logCompactionInstantTime);
     HoodieWriteMetadata<T> writeMetadata = table.logCompact(context, logCompactionInstantTime);
     HoodieWriteMetadata<O> logCompactionMetadata = convertToOutputMetadata(writeMetadata);
-    if (shouldComplete && logCompactionMetadata.getCommitMetadata().isPresent()) {
```

Comment:
Is `logCompactionMetadata.getCommitMetadata()` always present now?

Reply:
yes